docs/source/fitting/fitfunctions/CriticalPeakRelaxationRate.rst

```diff
@@ -0,0 +1,42 @@
+.. _func-CriticalPeakRelaxationRate:
+
+==========================
+CriticalPeakRelaxationRate
+==========================
+
+.. index:: CriticalPeakRelaxationRate
+
+Description
+-----------
+
+The Critical Peak Relexation Rate is defined as:
+
+.. math:: y = \frac{S_c}{(|x - T_c|)^a} + B_g
```

Comment:
we can write this using the "heavyside function" https://en.wikipedia.org/wiki/Heaviside_step_function which is usually written as a capital theta. So we could put:

B_1\Theta(x<T_c) + B_2\Theta(x>=T_c)

Reply:
Do I replace the whole of this with B_1\Theta(x<T_c) + B_2\Theta(x>=T_c)? or just part? and what are B_1 and B_2?